server/manager/src/main/java/org/apache/accumulo/manager/TabletGroupWatcher.java

```diff
@@ -1353,4 +1425,28 @@ private static void markDeadServerLogsAsClosed(WalStateManager mgr,
     }
   }
 
+  @VisibleForTesting
+  protected static class HighTablet {
+    private final KeyExtent extent;
+    private final boolean merged;
+    private final boolean prevRowColumn;
+
+    public HighTablet(KeyExtent extent, boolean merged, boolean prevRowColumn) {
+      this.extent = Objects.requireNonNull(extent);
+      this.prevRowColumn = prevRowColumn;
+      this.merged = merged;
+    }
+
+    public boolean isMerged() {
+      return merged;
+    }
+
+    public KeyExtent getExtent() {
+      return extent;
+    }
+
+    public boolean hasPrevRowColumn() {
```

Comment:
This name is confusing, it seems to be related to either one of the following :
 * There was a single tablet in the merge range when the merge marker was set
 * Delete tablets still needs to be done